src/moaicore/MOAIFreeTypeFont.cpp

```diff
@@ -590,7 +590,7 @@ USRect MOAIFreeTypeFont::DimensionsOfLine(cc8 *text, float fontSize, FT_Vector *
 	// Gather up the positions of each glyph
 	FT_Int penX = 0, penY = 0;
 	
-	for (size_t n = 0; n < maxGlyphs; ) {
+	for (size_t n = 0; numGlyphs < maxGlyphs; numGlyphs++) {
```

Comment:
This for loop feels really weird. I'm seeing how trying to clean it up expands into a much larger task. How would you feel about just switching it to a `while` loop until that cleanup happens, since it might make it a little more readable?

Reply:
+1

Does the value of `n` ever change in this loop?